src/client/components/draft/paragraph/test/paragraph.test.tsx

```diff
@@ -25,7 +25,7 @@ jest.mock("../../../rich_text/utils/text_selection", () => ({
 describe("Paragraph", () => {
   let getSelection
 
-  const getWrapper = passedProps => {
+  const getWrapper = (passedProps = props) => {
```

Comment:
Given that every call to `getWrapper` passes `props`, what was your motivation to give `passedProps` a default value?

Reply:
Oooh forgot one step, to stop passing props to `getWrapper` -- this has been updated.